pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MultistageGroupByExecutor.java

```diff
@@ -70,16 +70,15 @@ public class MultistageGroupByExecutor {
   // because they use the zero based integer indexes to store results.
   private final GroupIdGenerator _groupIdGenerator;
 
-  public MultistageGroupByExecutor(
-      int[] groupKeyIds,
-      AggregationFunction[] aggFunctions,
-      int[] filterArgIds,
-      int maxFilterArgId,
-      AggType aggType,
-      boolean leafReturnFinalResult,
-      DataSchema resultSchema,
-      Map<String, String> opChainMetadata,
-      @Nullable PlanNode.NodeHint nodeHint) {
+  public MultistageGroupByExecutor(int[] groupKeyIds,
+                                   AggregationFunction[] aggFunctions,
+                                   int[] filterArgIds,
+                                   int maxFilterArgId,
+                                   AggType aggType,
+                                   boolean leafReturnFinalResult,
+                                   DataSchema resultSchema,
+                                   Map<String, String> opChainMetadata,
+                                   @Nullable PlanNode.NodeHint nodeHint) {
```

Comment:
Could we keep the former style?

```
  public MultistageGroupByExecutor(
      int[] groupKeyIds,
      AggregationFunction[] aggFunctions,
      int[] filterArgIds,
      int maxFilterArgId,
      AggType aggType,
      boolean leafReturnFinalResult,
      DataSchema resultSchema,
      Map<String, String> opChainMetadata,
```

I know there is no style restriction here, but this way is closer to what both Kotlin and Scala do in their default styles.

Reply:
Sure, I thought you requested the other one.